website/docs/cli/commands/describe/describe-affected.md

```diff
@@ -0,0 +1,112 @@
+---
+title: atmos describe affected
+sidebar_label: affected
+sidebar_class_name: command
+id: affected
+description: This command produces a list of the affected Atmos components and stacks given two Git commits.
+---
+
+:::note Purpose
+Use this command to show a list of the affected Atmos components and stacks given two Git commits.
+:::
+
+<br/>
+
+:::info
+For the first commit, the command assumes that the repo root is a Git checkout.
+
+The second commit is specified on the command line by using the `--ref` or `--sha` flags.
+
+If the flags are not provided, it will clone the HEAD of the default branch.
+:::
+
+## Usage
+
+```shell
+atmos describe affected [options]
+```
+
+<br/>
+
+:::tip
+Run `atmos describe affected --help` to see all the available options
+:::
+
+## Examples
+
+```shell
+atmos describe affected
+atmos describe affected --verbose=true
+atmos describe affected --ref refs/heads/main
+atmos describe affected --ref refs/heads/my-new-branch --verbose=true
+atmos describe affected --ref refs/heads/main --format json
+atmos describe affected --ref refs/tags/v1.16.0 --file affected.yaml --format yaml
+atmos describe affected --sha 3a5eafeab90426bd82bf5899896b28cc0bab3073 --file affected.json
+atmos describe affected --sha 3a5eafeab90426bd82bf5899896b28cc0bab3073
+```
```

Comment:
Ideally, we should break each of these apart and explain what it does. 

for example, 

~~~
This command will describe all affected stacks relative to the default branch
```
atmos describe affected
```

This command will describe all affected stacks relative to the main branch provide verbose output including every step performed to arrive at the conclusion. 

```
atmos describe affected --verbose=true
```

~~~